mkt/reviewers/templates/reviewers/moderatelog_detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends "reviewers/base.html" %}
+
+{% block breadcrumbs %}
+{{ reviewers_breadcrumbs(
+      items=[(url('reviewers.apps.moderatelog'),_('Moderated Review Log')),
+      (None, _('Log Details'))]) }}
+{% endblock %}
+
+{% block content %}
+  <div class="featured">
+    <div class="featured-inner featured-inner-padded">
+      <h2>{{ _('Log details') }}</h2>
+      <p>{{ log.to_string('editor') }}</p>
+      {% if log.details %}
+        <dl>
+          <dt>{{ _('Moderation date') }}</dt>
+          <dd>{{ log.created|babel_datetime }}</dd>
+          <dt>{{ _("App title") }}</dt>
+          <dd>{{ log.details.addon_title }}</dd>
+          <dt>{{ _("Review title") }}</dt>
+          <dd>{{ log.details.title }}</dd>
+          <dt>{{ _("Review text") }}</dt>
+          <dd>{{ log.details.body }}</dd>
+        </dl>
+        {% if log.details.is_flagged %}
+          <p><strong>{{ _("Review was flagged prior to moderation.") }}</strong></p>
```

Comment:
there's a mix of ' and " in this template for translated strings, I think we aim to stick to '?